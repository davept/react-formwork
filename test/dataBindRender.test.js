import React from 'react';
import DataBindForm from './forms/dataBindForm';
import renderer from 'react-test-renderer';

test('Test a simple render', () => {
    const component = renderer.create(
        <DataBindForm />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
