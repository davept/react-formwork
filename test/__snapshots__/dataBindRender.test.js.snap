// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test a simple render 1`] = `
<form>
  <button
    type="submit"
  >
    Submit
  </button>
  <pre>
    {
  "name": "David",
  "email": "a@b.c",
  "jobTitle": "Developer"
}
  </pre>
</form>
`;
